.github/workflows/build-terraform-oss.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: build_terraform
+
+# This workflow is intended to be called by the build workflow for each Consul
+# binary that needs to be built and packaged. The crt make targets that are
+# utilized automatically determine build metadata and handle building and
+# packing consul.
```

Comment:
Oops, copy paste problems! Also not quite accurate since we only build the one binary for Terraform.

```suggestion
# This workflow is intended to be called by the build workflow. The crt make
# targets that are utilized automatically determine build metadata and
# handle building and packing Terraform.
```